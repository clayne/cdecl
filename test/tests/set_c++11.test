cdecl @ /dev/null @ @ set c++11 @ 0
