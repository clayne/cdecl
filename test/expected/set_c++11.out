  noalt-tokens
  nodebug
  noeast-const
  noexplain-by-default
    explicit-ecsu=su
  noexplicit-int
  nographs
    lang=C++11
    prompt
    semicolon
